### ISSUE TITLE
Documentation missing to use in combination with supabase for decoupled database?

### ISSUE THREAD

**Issue by argo-uka** [supabase/postgres#477]
# Improve documentation

## Describe the problem

`supabase/all` --> the main repo which runs everything (kong, postgrest, realtime....)
`supabase/postgres` --> this repo

It is not quite clear how to use this repo to use it with `supabase/all` for decoupled postreges db. This is the most featureful Postgres docker image, yet it is quite confusing how to connect it with `supabase/all`. The schema migrations files doesn't work, For example, the `realtime` schema has become `_realtime` ? 

I still don't know how to merge all to decouple correctly. New to docker so it is quite a journey, I'm happy to learn. But there are a lot of moving parts and it is confusing.

## Describe the improvement

- Please, provide a single docker-compose file for `supabase/postgres`, with the correct env vars to make it easily reachable from `supabase/all` compose file. In the current situation in `supabase/postgres`, in default branch there are 3 different `docker-compose` file each one ponting to different `supabase/postgres`. It is not quite clear which one is to run.
- Please remove the db service in the `supabase/all` docker-compose file.
- Please make the decoupled postgres the default getting started guide. 

Hope to not sound arrogant.

**Comment by cristian-milea** [comment]
I find myself in the same situation. I've been struggling for 3 days now to have a decoupled Supabase db and it's very hard to follow instructions. It feels like some steps are missing and I have to come up with solutions.

If it is recommended to decouple the database, why is it so hard to configure it? :))

Love this project, but **please** make it easier to understand for newcomers.

**Comment by rienheuver** [comment]
I'm in this situation now as well. Trying to decouple the database and running the self-hosted Supabase, but so far no luck getting postgres up and running. Documentation on this repo is unclear to me to know how to get it running. I want to run it without Docker so my data is not gone when the machine is restart. How does this repo help me with that?

**Comment by olirice** [comment]
If you want to run the supabase docker image stand-alone you can do that with
```
docker run -it --name supag -p 5535:5432 -d -e POSTGRES_DB=postgres -e POSTGRES_PASSWORD=local_pw -e POSTGRES_USER=postgres -d supabase/postgres:15.1.1.86
```
but at this time we don't support separating the database from the rest of the docker compose file in supabase/supabase

> I want to run it without Docker so my data is not gone when the machine is restart

By default the supabase CLI spins up the postgres docker image with a volume attached to your local disk. The data survives restarts. If you're seeing something different please open an issue on supabase/cli
